packages/build/tests/edge_functions/snapshots/tests.js.md

```diff
@@ -1,4 +1,4 @@
-# Snapshot report for `packages/build/tests/edge_functions/tests.js`
+# Snapshot report for `tests/edge_functions/tests.js`
```

Comment:
I think this happens when you run the test command from the package directory and not the root. 🤷🏻